source/installation-guide/wazuh-indexer/step-by-step.rst

```diff
@@ -153,6 +153,12 @@ Deploying certificates
 
   .. include:: /_templates/installations/indexer/common/deploy_certificates.rst
 
+
+#. Create username and password for the indexer
+
+  .. include:: /_templates/installations/indexer/common/create_username_password.rst
```

Comment:
Move this to the end of `_templates/installations/indexer/common/deploy_certificates.rst` for the correct format